packages/discord.js/src/structures/ThreadChannel.js

```diff
@@ -360,13 +363,11 @@ class ThreadChannel extends Channel {
    * @example
    * // Set the thread's auto archive time to 1 hour
    * thread.setAutoArchiveDuration(60)
-   *   .then(newThread => {
-   *     console.log(`Thread will now archive after ${newThread.autoArchiveDuration} minutes of inactivity`);
-   *    });
+   *   .then(newThread => console.log(`Thread will now archive after ${newThread.autoArchiveDuration} minutes of inactivity`));
```

Comment:
This line is too long, and the linter will complain about it. That's why it was setup the way it was beforehand